docs/reference/tools/pd-control.md

```diff
@@ -98,48 +98,119 @@ Use this command to view or modify the configuration information.
 Usage:
 
 ```bash
->> config show                                // Display the config information of the scheduler
+>> ./bin/pd-ctl config show all   // Display all config information of the scheduler
 {
-  "max-snapshot-count": 3,
-  "max-pending-peer-count": 16,
-  "max-merge-region-size": 20,
-  "max-merge-region-keys": 200000,
-  "split-merge-interval": "1h0m0s",
-  "patrol-region-interval": "100ms",
-  "max-store-down-time": "1h0m0s",
-  "leader-schedule-limit": 4,
-  "region-schedule-limit": 4,
-  "replica-schedule-limit":8,
-  "merge-schedule-limit": 8,
-  "tolerant-size-ratio": 5,
-  "low-space-ratio": 0.8,
-  "high-space-ratio": 0.6,
-  "disable-raft-learner": "false",
-  "disable-remove-down-replica": "false",
-  "disable-replace-offline-replica": "false",
-  "disable-make-up-replica": "false",
-  "disable-remove-extra-replica": "false",
-  "disable-location-replacement": "false",
-  "disable-namespace-relocation": "false",
-  "schedulers-v2": [
-    {
-      "type": "balance-region",
-      "args": null,
-      "disable": false
+  "client-urls": "http://127.0.0.1:2379",
```

Comment:
Maybe we can just give a few samples then say `// ... Omitted`? It seems too long and our config changes with new versions anyways, saves us updating it each time.

Reply:
It's a valid suggestion - the PD control reference is not very friendly to read and update as well. We can work something out after 3.0 release.